<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Фінанси</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::FormatTextBold"/>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #fef6e4; /* Кремовий фон */
    color: #000000;           /* Чорний текст за замовчуванням */
    font-family: &quot;Arial&quot;, sans-serif;
    font-size: 14px;
}

/* Всі тексти в таблиці */
QTableView, QTableView QHeaderView::section {
    color: #000000; /* Чорний текст */
    background-color: #ffffff; /* Білий фон таблиці */
    gridline-color: #9b2226;   /* Червоні лінії таблиці */
}

/* Заголовки таблиці */
QHeaderView::section {
    background-color: #f28482; /* Пастельно-червоний фон заголовків */
    font-weight: bold;
    border: 1px solid #9b2226;
    color: #000000; /* Чорний текст заголовків */
}

/* Кнопки */
QPushButton {
    background-color: #84a59d; /* Ніжно-зелений фон */
    color: #000000;           /* Чорний текст кнопок */
    border: 2px solid #006d77;
    border-radius: 5px;
    padding: 5px;
}

QPushButton:hover {
    background-color: #f6bd60; /* Жовтий фон при наведенні */
    color: #000000;           /* Чорний текст кнопок при наведенні */
}

/* Поля вводу */
QLineEdit, QDateEdit, QDoubleSpinBox {
    background-color: #f7e8d4; /* Кремовий фон полів вводу */
    color: #000000;           /* Чорний текст */
    border: 1px solid #9b2226;
    border-radius: 3px;
    padding: 3px;
}

/* Лейбли з білим текстом */
QLabel {
    color: #000000; /* Білий текст для лейблів */
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>СУМА</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dateEdit">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2024</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ОПИС</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ДАТА ТРАНЗАКЦІЇ</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="prefix">
         <string/>
        </property>
        <property name="suffix">
         <string>грн</string>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>99999999.989999994635582</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Видалити</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Довати </string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
